farge

```diff
@@ -31,23 +31,25 @@ NOTIFICATION=false
 EXPIRE_TIME=120000
 
 show_help() {
-    echo "usage: farge [OPTIONS...]"
-    echo " "
-    echo "┌─┐┌─┐┬─┐┌─┐┌─┐"
-    echo "├┤ ├─┤├┬┘│ ┬├┤ "
-    echo "┴  ┴ ┴┴└─└─┘└─┘"
-    echo "Click on a pixel on your screen and show its color value"
-    echo ""
-    echo "optional arguments:"
-    echo "   --help                show this help message and exit"
-    echo "   --no-preview          dont show the selected color in a preview window"
-    echo "   --no-color-code       dont write the color code on the preview window"
-    echo "   --rgb                 show the color code as RGB instead of HEX"
-    echo "   --image-viewer        image viewer to use to show the preview (feh, mpv, sxiv)"
-    echo "   --notify              send the color code as notification, where the icon is the selected color"
-    echo "   --expire-time         amount of time the notification should be visible, in milliseconds"
-    echo "   --stdout              output the color value to stdout"
-    echo "   --version             show the version of farge you are using"
+    cat <<EOF
+usage: farge [OPTIONS...]
+
+┌─┐┌─┐┬─┐┌─┐┌─┐
+├┤ ├─┤├┬┘│ ┬├┤
+┴  ┴ ┴┴└─└─┘└─┘
+Click on a pixel on your screen and show its color value
+
+optional arguments:
+   --help (-h)           show this help message and exit
+   --version (-v)        show the version of farge you are using
+   --no-preview          dont show the selected color in a preview window
+   --no-color-code       dont write the color code on the preview window
+   --rgb                 show the color code as RGB instead of HEX
+   --image-viewer        image viewer to use to show the preview (feh, mpv, sxiv)
+   --notify              send the color code as notification, where the icon is the selected color
+   --expire-time         amount of time the notification should be visible, in milliseconds
+   --stdout              output the color value to stdout
+EOF
```

Comment:
use cat and a heredoc to reduce commands run and look a lot cleaner (also add `-h` and `-v`)